jsons/manifesto/it.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title":"Manifesto per l'Artigianato del Software",
+  "raising_the_bar":"Alzare l'asticella.",
+  "introduction":"Come aspiranti Artigiani del Software, stiamo alzando l'asticella dello sviluppo professionale del software praticandolo e aiutando gli altri a imparare il mestiere.  Attraverso questo lavoro siamo arrivati a dare valore a:",
+  "not_only":"Non solo ",
+  "but_also":"ma anche ",
+  "working_software":"software funzionante,",
+  "well_crafted_software":"software ben fatto",
+  "responding_to_change":"rispondere al cambiamento,",
+  "steadily_adding_value":"valore aggiunto costante",
```

Comment:
Agreed! I prefer the first one "regolarmente" because according to the boy scout rule explained by uncle Bob, we check-in the code better than when it was checked out and we do this actually "regolarmente"

Reply:
Yes, good point